src/SarifCli/JPointerStringExtensions.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.CodeAnalysis.Sarif.Cli
+{
+    // TODO: These methods belong in the Microsoft.Json.Pointer library.
```

Comment:
Done. Filed [https://github.com/Microsoft/sarif-sdk/issues/512](https://github.com/Microsoft/sarif-sdk/issues/512), "Move JPointerExtensions from SarifCli into Microsoft.Json.Pointer library", and added comment.

---
In reply to: [77238851](https://github.com/Microsoft/sarif-sdk/pull/501#discussion_r77238851) [](ancestors = 77238851)